src/test/scala/is/hail/io/ImportMatrixSuite.scala

```diff
@@ -6,62 +6,118 @@ import is.hail.check.Gen
 import is.hail.check.Prop.forAll
 import is.hail.expr._
 import is.hail.expr.types._
+import is.hail.table.Table
 import is.hail.variant.{MatrixTable, VSMSubgen}
 import org.apache.spark.SparkException
 import org.testng.annotations.Test
 import is.hail.utils._
 
 class ImportMatrixSuite extends SparkSuite {
 
+  val genMatrix = VSMSubgen(
```

Comment:
is there a more descriptive name for this? Is this just the types of matrices that the importer currently supports? Could the rows, cols, row-annos, and col-annos be more generic?